src/app/views/App.jsx

```diff
@@ -206,13 +196,15 @@ const App = () => {
       setError(createProjectError);
       return;
     }
+    setError('');
     console.log(createProjectResult);
     // if all goes ok, then set the state to display the code and signIn with the provided contract
     const isSignedInSuccessfully = await signIn(contractAddress);
     if (!isSignedInSuccessfully) {
       setError(`Cannot Sign with with contract address ${contractAddress}`);
       return;
     }
+    setError('');
```

Comment:
2 set errors?

Reply:
`setError('')` cleans up previous errors. e.g. when you try to sign up multiple times